internal/api/sms_provider/sms_provider.go

```diff
@@ -40,3 +43,7 @@ func GetSmsProvider(config conf.GlobalConfiguration) (SmsProvider, error) {
 		return nil, fmt.Errorf("sms Provider %s could not be found", name)
 	}
 }
+
+func IsValidMessageChannel(channel string, config conf.GlobalConfiguration) bool {
+	return (channel == SMSProvider || channel == WhatsappProvider) && config.Sms.Provider == "twilio"
```

Comment:
I feel like these complex conditions can be more readable with a switch statement?